pep-0582.rst

```diff
@@ -5,12 +5,13 @@ Last-Modified: $Date$
 Author: Kushal Das <mail@kushaldas.in>, Steve Dower <steve.dower@python.org>,
         Donald Stufft <donald@stufft.io>, Nick Coghlan <ncoghlan@gmail.com>
 Discussions-To: https://discuss.python.org/t/pep-582-python-local-packages-directory/963/
-Status: Draft
+Status: Rejected
 Type: Standards Track
 Topic: Packaging
 Content-Type: text/x-rst
 Created: 16-May-2018
 Python-Version: 3.12
```

Comment:
```suggestion
Python-Version: 3.12
Post-History: `01-Mar-2019 <https://discuss.python.org/t/pep-582-python-local-packages-directory/963>`__,
```

Adding this for completeness; I couldn't find any other canonical discussion threads for this searching Discourse and the Python-Dev mailing list.